examples/tf/vpg_cartpole.py

```diff
@@ -1,27 +1,24 @@
 """
 This is an example to train a task with VPG algorithm.
 
-Here it runs CartpoleEnv with 100 iterations.
+Here it runs CartPole-v1 environment with 100 iterations.
 
 Results:
-    AverageReturn: 1000
-    RiseTime: itr 60
+    AverageReturn: 100
+    RiseTime: itr 16
 """
-
 from garage.baselines import LinearFeatureBaseline
-from garage.envs import normalize
-from garage.envs.box2d import CartpoleEnv
 from garage.experiment import run_experiment
 from garage.tf.algos import VPG
 from garage.tf.envs import TfEnv
-from garage.tf.policies import GaussianMLPPolicy
+from garage.tf.policies import CategoricalMLPPolicy
 
 
 def run_task(*_):
     """Wrap VPG training task in the run_task function."""
-    env = TfEnv(normalize(CartpoleEnv()))
+    env = TfEnv(env_name='CartPole-v1')
```

Comment:
`env = TfEnv(env_name="CartPole-v1")`

Reply:
Thanks.

I have been meaning to update the style guide to specify single-quotes strings.

Mainly because (1) it seems that single quotes are more popular in Python than double quotes (e.g. https://www.kennethreitz.org/essays/if-i-could-amend-pep-8), and (2) as a result, there's automated tooling for single quotes but not double quotes https://github.com/rlworkgroup/garage/blob/master/.pre-commit-config.yaml#L62